samples/snippets/src/main/java/pubsublite/PublishWithCustomAttributesExample.java

```diff
@@ -64,15 +67,23 @@ public static void publishWithCustomAttributesExample(
     // Start the publisher. Upon successful starting, its state will become RUNNING.
     publisher.startAsync().awaitRunning();
 
-    String message = "message-with-custom-attributes";
+    // Prepare the message data as a byte string.
+    String messageData = "message-with-custom-attributes";
+    ByteString data = ByteString.copyFromUtf8(messageData);
+
+    // Prepare a protobuf-encoded event timestamp for the message.
+    Instant now = Instant.now();
+    String eventTime =
+        MessageTransforms.encodeAttributeEventTime(
+            Timestamp.newBuilder().setSeconds(now.getEpochSecond()).build());
```

Comment:
use https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/util/Timestamps

`Timestamps.fromMillis(now.toEpochMilli())`


toEpochMilli()
--





toEpochMilli()
--

Reply:
That's better. Udpated.